ostree/gen-fixtures.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+DIR=""
+TREE=$(mktemp -d /tmp/tree.XXXXX)
+USAGE="
+Usage: gen-fixtures.sh <output-dir>
+
+Generate an OSTree repository with (2) branches:
+ - rawhide
+ - stable
+
+Each branch will have (3) commits for versions:
+ - 1.0
+ - 1.1
+ - 1.2
+
+The OSTree repository will be created in <output-dir>.
+The directory will be be created.
+"
+
+show_help()
+{
+  echo "${USAGE}"
+}
+
+_mkdir()
+{
+  if [ -e ${DIR} ]; then
+    if [ ! -z "$(ls -A ${DIR})" ]; then
+      echo 1>&2 ""
+      echo 1>&2 "${DIR} must be empty."
+      echo 1>&2 ""
+      exit 1
+    fi
+  else
+    mkdir -p ${DIR}
+  fi
+}
+
+gen_tree()
```

Comment:
This function copies this script into `/tmp/tree.XXXXX/`. What function does that serve?